### PR TITLE
Fix extent for repeated 300 with multiple Subfields

### PR DESCRIPTION
**Pull request by maipet**
Adjust replacement of brackets

### CODE REVIEW

File: src/test/resources/alma-fix/990198383780206441.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "id" : "http://id.loc.gov/vocabulary/iso639-2/ger",
     "label" : "Deutsch"
   } ],
-  "extent" : "1 CD : digital ; 12 cm, 100 gr ; 1 Booklet [(2)] S (1 CD (75 04 Min. ; 12 cm",
+  "extent" : "1 CD : digital ; 12 cm, 100 gr ; 1 Booklet [(2)] S (1 CD (75 04 Min.) ; 12 cm",
   "hasItem" : [ {
```
Comment (dr0i) [inline src/test/resources/alma-fix/990198383780206441.json:75]: still a closing bracket missing

**Comment by maipet** [comment]
I didn't get it 100%, maybe @TobiasNx can help?

**Comment by dr0i** [comment]
Could solve it by print-debugging - i.e. insert some identifiable messages in (here) `replace` functions and let  the tests rebuild the data anew with `mvn clean install -DskipTests=false  -DgenerateTestData=true`. Then `git diff` and search for the messages. If you know which statements should/could provide the missing bracket (or which variable is involved and trace that variable) it's not so far from try-and-error-until-resolved.

**Comment by TobiasNx** [comment]
I did not find a corresponding issue for this. We should add issues for upcoming changes in the scrips if they result from errors to better document our progress.

**Pull request merged**